trillian/integration/copier.go

```diff
@@ -153,14 +153,22 @@ func NewCopyChainGeneratorFromOpts(ctx context.Context, client *client.LogClient
 		StartIndex:    startIndex,
 	}
 	certFetcher := scanner.NewFetcher(client, &fetchOpts)
-	go certFetcher.Run(ctx, func(batch scanner.EntryBatch) {
-		generator.processBatch(batch, generator.certs, ct.X509LogEntryType)
-	})
+	go func() {
```

Comment:
nit: I think you could put `certFetcher := ...` and `precertFetcher := ...` into their respective `func()`s

Reply:
done.